charts/secrets-injector/README.md

```diff
@@ -0,0 +1,38 @@
+# 1Password Kubernetes Secrets Injector Helm chart
+Installing the Helm Chart with default configurations will deploy 1Password Kubernetes Secrets Injector in your default Namespace.
+
+## Installation
+```
+helm install --generate-name 1password/secrets-injector
+```
+
+## Usage
+Please follow [Secrets Injector documentation](https://github.com/1Password/kubernetes-secrets-injector#1password-secrets-injector-for-kubernetes)
+
+
+## Configuration Values
+The 1Password Kubernetes Secrets Injector Helm chart offers many configuration options for deployment. Please refer to the list below for information on what configuration options are available as well as what the default configuration options are.
+
+[From the Official Helm Install Guide](https://helm.sh/docs/helm/helm_install/#helm-install):
+
+>To override values in a chart, use either the '--values' flag and pass in a file or use the '--set' flag and pass configuration from the command line, to force a string value use '--set-string'. In case a value is large and therefore you want not to use neither '--values' nor '--set', use '--set-file' to read the single large value from file.
+
+For example: 
+```bash
+$ helm install -f myvalues.yaml injector ./secrets-injector
+```
+
+or 
+
+```bash
+$ helm install --set injector.applicationName=injector injector ./secrets-injector
+```
+
+### Values
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| injector.applicationName | string | `"secrets-injector"` | The name of 1Password Kubernetes Secrets Injector Application |
```

Comment:
should we add a value for configuring the version of the injector here as well in case you don't want the latest?

Reply:
It uses the chart AppVersion
```
image: {{ .Values.injector.imageRepository }}:{{ .Chart.AppVersion }}
```
So, we always use the explicit version.